packages/trader/src/AppV2/Components/ContractDetailsFooter/contract-details-footer.tsx

```diff
@@ -0,0 +1,84 @@
+import { Button } from '@deriv-com/quill-ui';
+import { TContractInfo, getCardLabels, isMultiplierContract, isValidToCancel, isValidToSell } from '@deriv/shared';
+import { useStore } from '@deriv/stores';
+import { getRemainingTime } from 'AppV2/Utils/helper';
+import { observer } from 'mobx-react';
+import React from 'react';
+
+type ContractInfoProps = {
+    contract_info: TContractInfo;
+};
+
+const ContractDetailsFooter = observer(({ contract_info }: ContractInfoProps) => {
+    const {
+        bid_price,
+        currency,
+        contract_id,
+        cancellation: { date_expiry: cancellation_date_expiry } = {},
+        profit,
+        contract_type,
+    } = contract_info;
+
+    const { contract_replay, common } = useStore();
+    const { server_time } = common;
+    const { onClickCancel, onClickSell, is_sell_requested } = contract_replay;
+    const is_valid_to_sell = isValidToSell(contract_info);
+    const is_valid_to_cancel = isValidToCancel(contract_info);
+    const is_multiplier = isMultiplierContract(contract_type);
+
+    const cardLabels = getCardLabels();
+    const bidDetails = !is_valid_to_cancel ? `@${bid_price} ${currency}` : '';
```

Comment:
please use `formatMoney()` helper
to avoid this issue with 0.0 value instead of 0.00 value:
<img width="480" alt="Screenshot 2024-06-13 at 2 25 07 PM" src="https://github.com/binary-com/deriv-app/assets/103177211/49fa0fad-7af4-4df9-9f33-3431e8a90970">

Reply:
Done thanks

Reply:
formatMoney function is deprecated in deriv/shared .